test/test_robot_wrapper.py

```diff
@@ -407,3 +417,119 @@ def test_fetch_robot_wrapper(fixed_base):
                 "test_fetch_robot_wrapper__fixed_base=" + str(fixed_base),
                 open_vid=True,
             )
+
+
+@pytest.mark.skipif(
+    not osp.exists("data/robots/hab_franka"),
```

Comment:
Also, this should be the path which is created from downloading [this asset](https://github.com/facebookresearch/habitat-sim/blob/83fe46b9695bc9a8d4187f3d739ddbd1b2889619/src_python/habitat_sim/utils/datasets_download.py#L129). I think that's actually `"data/robots/franka_panda`

Once you update that, check that this test isn't skipped on CI. It should be getting the robot assets from the downloader util already.

Reply:
Done!